### ISSUE TITLE
Feature Request: Support Jackson

### ISSUE THREAD

**Issue by dcr-stripe** [stripe/stripe-java#1344]

**Comment by pizzi80** [comment]
maybe it's better to move to standard Java Json Binding (JSR 367)

[https://javaee.github.io/jsonb-spec/](https://javaee.github.io/jsonb-spec/)

**Comment by dcr-stripe** [comment]
Good idea @pizzi80 - I think in general we want to support a more pluggable JSON infrastructure, so this could certainly fall under that. I don't think we necessarily want to only support one library since we'll have the same problem we have today where folks are pulling in a different library through their app framework and don't want to need two JSON libraries.

**Comment by pizzi80** [comment]
Yes I know... actually this is the biggest error of Java as a language in the last decade
Now they've finally fixed it with Json Binding, so probably frameworks will slowly move to jsonb
but it will require another decade :rofl: 

I hope that develop a json agnostic library will not require too much effort for you
